spec/pane-spec.coffee

```diff
@@ -153,6 +153,89 @@ describe "Pane", ->
       pane.activateItem(pane.itemAtIndex(1))
       expect(observed).toEqual [pane.itemAtIndex(1)]
 
+    fdescribe "when the `pending` option is true", ->
```

Comment:
Is `fdescribe` a new helper that I haven't heard of yet? I only know of `describe`